_locales/en/messages.json

```diff
@@ -331,7 +343,10 @@
     },
     "defaultChangelogAction": {
         "message": "Changelog"
-    },    
+    },
+    "defaultChangelogActionClose": {
+        "message": "Close"
```

Comment:
We already have a generic message `close`. We should probably rather use that one.

Reply:
Changed ;)